pkg/upgrade/upgrader/jiva_upgrade.go

```diff
@@ -584,6 +611,18 @@ func (j *jivaVolumeOptions) verify(pvLabel, openebsNamespace string) error {
 		}
 		return err
 	}
+	if currentVersion < "1.9.0" {
```

Comment:
It may be possible that the job dies after cleaning up old replica deployments. So when the job restarts, will it still know that it has to go and cleanup the pending target deployment. One thought is -- just call the j.cleanup anyway without version check, and that cleanup will only delete stale entries.

Reply:
If job dies after cleanup is successful it will not attempt to delete the deployment as it will not be able to find old deployment and the `replicaObj.name` variable will be empty and we check on that before deleting the deployment.